aks/redis/variables.tf

```diff
@@ -95,6 +95,12 @@ variable "azure_enable_monitoring" {
   default = true
 }
 
+variable "azure_zones" {
```

Comment:
Do we really need an input variable here? We could set it to all zones by default when the SKU allows it
ie when it's not Basic or Standard

Reply:
thats true could set the default to 1.2 only as per the dev deployment

Reply:
updated the default to 2 zones

Reply:
removed variable